src/main/java/games/strategy/triplea/ai/proAI/ProCombatMoveAI.java

```diff
@@ -629,7 +629,7 @@ private void removeTerritoriesWhereTransportsAreExposed() {
         }
 
         // Determine counter attack results for each transport territory
-        double totalEnemyTUVSwing = 0.0;
+        double totalEnemyTuvSwing = 0.0;
```

Comment:
`enemyTuvSwing` is perhaps better here instead of `totalEnemyTuvSwing`

Tuv = 'total unit value'
So the variable  'totalEnemyTuvSwing'  spelled out would be: total enemy total unit value swing